pod.j2

```diff
@@ -19,17 +19,24 @@ spec:
       resources:
         requests:
           cpu: 16
+          memory: 2G
         limits:
           cpu: 16
+          memory: 2G
       command: ["cyclictest"]
       args:
-        - --smp
         - --duration={{ duration }}
         - --histofall=US
         - --mlockall
-        - --priority=80
-        - --interval=200
+        - --priority=95
+        - --interval=1000
         - --distance=0
+        - --smi
+        - -a
+        - -t 16
+        - -n
+        - -h 30
```

Comment:
`-h       --histogram=US`: dump a latency histogram to stdout after the run US is the max latency time to be be tracked in microseconds. This option runs all threads at the same priority.
`-H       --histofall=US`: same as `-h` except with an additional summary column
Therefore `--histofall=US`, which we can set to `--histofall=30` instead.

Reply:
We can leave just the --histofall=US

Reply:
--histofall=US was my bug, see https://github.com/Avielyo10/cyclictest/commit/10fccb2689dced6a95d51e4d92ad1a61b63d5a57